lib/rails/generators/alchemy/install/files/application.html.erb

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%= @page.language_code %>">
```

Comment:
The `@page.language_code` is an attribute on the page table. It's set on page save. The `Language.current.code` also includes the country code, where the page language code is missing the country part. So, the language object is the truth. The page language attribute a kind of cached attribute that not always has the correct value. We should remove this someday, because it's confusing.